chrome/content/zotero/xpcom/itemTreeManager.js

```diff
@@ -137,13 +137,13 @@ class ItemTreeManager {
 	}
 
 	/**
-	 * Check if column option is valid atomically
+	 * Check if column option is valid. An atomic operation.
```

Comment:
Strictly speaking, an atomic operation is something that happens in a low-level code situation where memory is shared and multiple threads are accessing it, and the atomic operation changes the shared memory in such a way, that other threads never see an intermediate value, and as such you don't get a race condition. Traditionally semaphores have to change atomically.

I understand what you mean, because I've read this code, but after we come back to it in X years, the "An atomic operation" comment will just be confusing. Please just change this to an actual explanation of what we're doing here, i.e. we need to validate all column data keys at once to prevent duplicates.